evap/evaluation/management/commands/tools.py

```diff
@@ -47,3 +48,13 @@ def logged_call_command(stdout, *args, **kwargs):
     """Log execution of management command with all args."""
     stdout.write("Executing python manage.py " + " ".join(list(args) + [f"{a}={b}" for a, b in kwargs.items()]))
     call_command(*args, **kwargs)
+
+
+def subprocess_run_or_exit(command: list, stdout=None) -> None:
+    if stdout is not None:
+        stdout.write(f"Executing {' '.join(str(el) for el in command)}")
+
+    exit_code = subprocess.run(command, check=False).returncode
```

Comment:
Should we
```suggestion
    exit_code = subprocess.run(command, stdout=stdout, check=False).returncode
```
or does that not make sense?